src/apprt/gtk/key.zig

```diff
@@ -105,32 +105,25 @@ pub fn keyvalUnicodeUnshifted(
 /// This requires a lot of context because the GdkEvent
 /// doesn't contain enough on its own.
 pub fn eventMods(
-    widget: *c.GtkWidget,
     event: *c.GdkEvent,
     physical_key: input.Key,
     gtk_mods: c.GdkModifierType,
-    x11_xkb: ?*x11.Xkb,
+    app_platform: platform.AppState,
 ) input.Mods {
     const device = c.gdk_event_get_device(event);
 
-    var mods = mods: {
+    var mods = switch (app_platform.inner) {
```

Comment:
Seems a little icky to be accessing `.inner` maybe move this to a function called `platform.AppState.getMods()`?

Reply:
Agree we should try to contain all the switching on "platforms" in one place if possible.

Reply:
Yeah I tried to encapsulate that inside platform, but then we have to move `translateMods` into there too..